enterprise_access/apps/api/v1/views/content_assignments/assignments_admin.py

```diff
@@ -204,3 +204,32 @@ def cancel(self, request, *args, uuid=None, **kwargs):
             return Response(response_serializer.data, status=status.HTTP_200_OK)
         else:
             return Response(status=status.HTTP_422_UNPROCESSABLE_ENTITY)
+
+    def remind(self, request, *args, uuid=None, **kwargs):
```

Comment:
you need the same permission and DRF-action decorators as the `cancel` view has:
```python
@permission_required(CONTENT_ASSIGNMENT_ADMIN_WRITE_PERMISSION, fn=assignment_admin_permission_fn)
@action(detail=True, methods=['post'])
```